docs/source/install.rst

```diff
@@ -80,25 +80,17 @@ Create a ``do-configure`` script such as:
     EXTRA_ARGS=$@
 
     cmake \
-        -D CMAKE_INSTALL_PREFIX:PATH=$INSTALL_DIR \
-        -D CMAKE_BUILD_TYPE:STRING=DEBUG \
-        -D CMAKE_VERBOSE_MAKEFILE:BOOL=OFF \
-        -D BUILD_SHARED_LIBS:BOOL=OFF \
-        -D TPL_ENABLE_MPI:BOOL=ON \
-        -D TPL_ENABLE_Boost:BOOL=ON \
-        -D Boost_INCLUDE_DIRS:PATH=$BOOST_INCLUDE_DIR \
-        -D TPL_ENABLE_Libmesh:BOOL=OFF \
-        -D TPL_ENABLE_Netcdf:BOOL=OFF \
-        -D TPL_ENABLE_MOAB:BOOL=OFF \
-        -D TPL_ENABLE_BinUtils:BOOL=OFF \
-        -D Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES:BOOL=OFF \
+        -D CMAKE_BUILD_TYPE=Release \
+        -D TPL_ENABLE_MPI=ON \
+        -D TPL_ENABLE_BLAS=ON \
+        -D TPL_ENABLE_LAPACK=ON \
+        -D TPL_ENABLE_Boost=ON \
+        -D Trilinos_ENABLE_ALL_OPTIONAL_PACKAGES=OFF \
         -D Trilinos_EXTRA_REPOSITORIES="DataTransferKit" \
-        -D Trilinos_ENABLE_EXPLICIT_INSTANTIATION:BOOL=ON \
```

Comment:
Why remove this? People will try compile without ETI and will run away screaming.

Reply:
Because I did not want to bother giving the a list of types to instantiate.

Reply:
I added a note that the `do-configure` script might not be the most up-to-date place to look.

@aprokop Would you add the explicit instantiation lines to the script?

Reply:
@dalg24 Which script?

Reply:
"do-configure" in `docs/source/install.rst`

Reply:
Done.